conanfile.py

```diff
@@ -43,11 +43,12 @@ def source(self):
         # This small hack might be useful to guarantee proper /MT /MD linkage
         # in MSVC if the packaged project doesn't have variables to set it
         # properly
+        cmake_project = "project(Fruit VERSION %s LANGUAGES CXX)" % (self.version,)
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
```

Comment:
Instead of using replace_in_file, can we add another if to the CMakeLists.txt file, as in the other PR?

Reply:
Fixed in my last commit: 6d8bcbdd77352656446a496b4a4f86e459dbb0db

However, probably conflict with my other PR.

Reply:
Thanks!